bin/spice/pkg/github/runtime_release.go

```diff
@@ -114,9 +126,17 @@ func get_ai_accelerator() (string, bool) {
 		if err != nil {
 			slog.Error("checking for CUDA device", "error", err)
 		}
-		if version != nil {
-			return "cuda_" + *version, true
+
+		if version == nil {
+			return "", false
+		}
+
+		if !checkCudaVersionSupported(*version) {
+			slog.Warn(fmt.Sprintf("GPU version is not supported. Spice will run but without GPU acceleration. Detected GPU version: '%s'", *version))
```

Comment:
```suggestion
			slog.Warn(fmt.Sprintf("GPU version ('%s') is not supported. Spice will run but without GPU acceleration.", *version))
```
Could be shorter

Reply:
I would word it like:
`Spice detected a GPU, but the GPU version (%s) is not supported for model acceleration. Spice will fallback to using the CPU to run models, which may impact performance.`

Reply:
Thank you